app/src/main/java/com/yeonae/chamelezone/view/home/HomeTabFragment.kt

```diff
@@ -180,11 +181,7 @@ class HomeTabFragment :
                 Manifest.permission.ACCESS_FINE_LOCATION
             ) != PackageManager.PERMISSION_GRANTED
         ) {
-            ActivityCompat.requestPermissions(
-                requireActivity(),
-                arrayOf(Manifest.permission.ACCESS_FINE_LOCATION),
-                LOCATION_REQUEST_CODE
-            )
+            requireContext().shortToast("위치 권한이 필요합니다.")
```

Comment:
```suggestion
            context?.shortToast(getString(R.string.some_text))
```
requireContext()에 ctrl을 누르고 마우스 왼쪽클릭을 하면 구현체가 나오는데요. 보면 try-catch로 되어있을 거에요 catch의 상황에 해당하는 exception handle을 추가하지 않는다면, 이 방법이(context - nullable type) 조금 더 안전한 것 같습니다. 
또 공동작업이니 만큼 리소스 관리를 지속적으로 놓치지 않고 하는게 좋겠어요

Reply:
넵 수정했습니다!
8cc0ac2